MPC/config/dcps_ts_defaults.mpb

```diff
@@ -31,7 +31,7 @@ project : dds_macros, taobaseidldefaults {
   // Also To avoid multiple unnecessary warnings, have tao_idl ignore
```

Comment:
```suggestion
  // Also to avoid multiple unnecessary warnings, have tao_idl ignore
```

This is my mistake, but it'd be great if you could fix it.